### PR TITLE
Adding CryptoGopniks tokens to neutron and stargaze

### PR DESCRIPTION
**Pull request by MedvedCrypto**

### CODE REVIEW

File: neutron/assetlist.json [hunk 1]
```diff
@@ -1354,6 +1354,37 @@
...
+    "display": "GOP",
+    "symbol": "GOP",
+    "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/gop.png"
+    },
+    "images": [
+        {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/gop.png",
+            "theme": {
+                "primary_color_hex": "#808080"
```
Comment (JeremyParish69) [inline neutron/assetlist.json:1384]: Shouldn't this be more of a black color rather than a medium grey?

File: stargaze/assetlist.json [hunk 2]
```diff
@@ -566,6 +566,56 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stargaze/images/vault.png"
         }
       ]
+    },
+    {
+      "description": "Liquid token STARS from NFT lending platform GopLend",
```
Comment (JeremyParish69) [inline stargaze/assetlist.json:571]: Sorry, I'm not quite understanding how this is derived. This is a liquid form of what position, exactly?

Comment (MedvedCrypto) [reply stargaze/assetlist.json:571]: Thanks for your attention, yes, this is a liquid form of lended STARS, i.e. like stSTARS from Stride (just not for staking)

This token is issued in exchange for the $STARS token when a user makes a deposit on the platform

**Comment by MedvedCrypto** [comment]
I've made a small edit to the description, it should be a little clearer now

File: stargaze/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply stargaze/assetlist.json]: Thanks for explaining. I was wondering if this aught to describe how it derives from STARS using traces[], but there is currently no trace type to describe liquid-lending (I presume this works kinda like bonds).

**Pull request merged**